api/v1alpha1/ippool_types.go

```diff
@@ -14,6 +14,31 @@ import (
 // disables net-operator's IPAM for that GatewayClass.
 const IPAMDisabledAnnotationKeyName = "netoperator.vmware.com/ipam-disabled"
 
+type IPPoolUsageLabelValue string
+
+const (
+	// IPPoolUsageLabelKeyName is the name of a label used to indicate how IP pools
+	// should be used. To create an affinity, you must create a NetworkInterface with a
+	// label matching the intended use. For example, if you create a NetworkInterface with
+	// a label matching netoperator.vmware.com/ipam-usage=vip, then net operator
+	// will only provision from IPPools matching that label falling back to the general
+	// pool if needed unless IPPoolUsageAnnotationStrictKeyName is set.
+	IPPoolUsageLabelKeyName = "netoperator.vmware.com/ipam-usage"
+
+	// IPPoolUsageAnnotationStrictKeyName indicates that an interface should not attempt
+	// to retrieve IPPools meant for general purpose consumption. For example, if "vip" is set,
+	// only IPPools matching the "vip" label will be used and "general" will not be used as a pool.
+	IPPoolUsageAnnotationStrictKeyName = "netoperator.vmware.com/ipam-usage/strict"
```

Comment:
I don't believe this key is valid b/c there can only be at most one slash: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

Reply:
TIL. Thanks!